packages/google_sign_in/google_sign_in/README.md

```diff
@@ -61,6 +61,20 @@ This plugin requires iOS 9.0 or higher.
 <!-- End of the Google Sign-in Section -->
 ```
 
+### iOS Authenticate with backend server
```

Comment:
Is there some documentation you can point us to for where this information comes from? Without context it's pretty hard to review this.

Reply:
No, I did'nt found any, it's why I'm doing this PR. This has been introduced in v4.5.0.

Here a comment mentioning it: https://github.com/flutter/flutter/issues/57945#issuecomment-758269872

The PR introducing it: https://github.com/flutter/plugins/pull/2116

The app example config: https://github.com/flutter/plugins/blob/f6495091d47070ec7e1aa456b236a04b674fc378/packages/google_sign_in/google_sign_in/example/ios/Runner/GoogleService-Info.plist#L41

This is the official doc for native iOS apps: https://developers.google.com/identity/sign-in/ios/offline-access

Reply:
@cyanglaz It looks like you reviewed the PR that added that example to the plist; could you review these instructions?